packages/components/components/Modal/Modal.tsx

```diff
@@ -0,0 +1,126 @@
+import React, { useCallback } from "react";
+import { Portal } from "react-portal";
+import Button from "../Button";
+import { Cross1Icon } from "@radix-ui/react-icons";
+import { styled } from "../../stitches.config";
+
+const CloseIcon = styled(Cross1Icon, {
+  position: "absolute",
```

Comment:
I don't understand why I need this, but the icon wouldn't appear without changing position. Also not sure how to make it thicker like in the designs?